<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sniff_dialog</class>
 <widget class="QDialog" name="sniff_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="sniff_browser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>551</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="filter_box">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>121</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Protocol Filter</string>
   </property>
   <widget class="QRadioButton" name="all_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="arp_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ip_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="icmp_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ICMP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="igmp_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IGMP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tcp_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="udp_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QListWidget" name="packet_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pkt_num">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>310</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11111/11111</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
